### PR TITLE
WAVE2-841 explore programs keyword filters

### PR DESCRIPTION
**Pull request by balrajb**

### CODE REVIEW

**Comment by dpfitzsi** [comment]
Navinder this is ready for peer review.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-exposed-form--ucgy-fgs-all-programs.tpl.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<?php if (!empty($q)): ?>
+    <?php
+    // This ensures that, if clean URLs are off, the 'q' is added first so that
+    // it shows up first in the URL.
+    print $q;
+    ?>
+<?php endif; ?>
+<div class="views-exposed-form">
+    <div class="views-exposed-widgets clearfix">
+        <?php foreach ($widgets as $id => $widget): ?>
```
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-exposed-form--ucgy-fgs-all-programs.tpl.php]: Can we please add few comments regarding this foreach loop.

Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-exposed-form--ucgy-fgs-all-programs.tpl.php]: added comments, however this specific loop is standard exposed tpl.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc [hunk 2]
```diff
@@ -933,8 +933,8 @@ function ucgy_fgs_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
-        'thesis-based' => 'Thesis based',
-        'course-based' => 'Course based',
+        'thesis-based' => ' Thesis based',
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: Is there a reason that a space is being added before the values?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/ucgy_fgs_all_programs.info [hunk 3]
```diff
@@ -7,7 +7,7 @@ dependencies[] = ctools
 dependencies[] = views
 dependencies[] = ucgy_fgs
 features[ctools][] = views:views_default:3.0
-;scripts[] = js/ucgy_fgs_all_programs.js
-;stylesheets[all][] = css/ucgy_fgs_all_programs.css
+scripts[] = js/ucgy_fgs_all_programs.js
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/ucgy_fgs_all_programs.info]: This appears to be adding in the global script again. Wasn't this moved to a hook_init as part of 868?

**Comment by balrajb** [comment]
Issues fixed mentioned in Lead review.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+<div class="<?php echo $classes; ?> filters">
+    <div class="row">
+        <div class="container">
+            <div class="row">
+                <form id="filters" role="form">
+                    <div class="col-md-3 vspacer-xs vspacer-sm degree">
+                        <p><?php print t('Filter by degree'); ?>:</p>
+                        <p>
+                            <button type="button"
+                                    class="btn btn-default masters"
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php]: This should be changed to the singular Master.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+                                    value="masters">Masters
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_all_programs/templates/views-view--ucgy-fgs-all-programs-solr.tpl.php]: Masters should be changed to Master here too.

**Comment by balrajb** [comment]
@dpfitzsi changed masters to master https://github.com/ucalgary/www.ucalgary/pull/473/commits/321c4b1968279780ac1228ee1343e61990bc7a3c

**Pull request merged**